networkx/generators/geometric.py

```diff
@@ -290,11 +290,14 @@ def geographical_threshold_graph(
 
     .. math::
 
-       (w_u + w_v)h(r) \ge \theta
+       (w_u + w_v)p(r) \ge \theta
```

Comment:
```suggestion
       (w_u + w_v)\textrm{p_dist}(r) \ge \theta
```

Reply:
Yeah that makes sense. Save the renaming for another PR